cardano-cli/src/Cardano/CLI/Byron/Delegation.hs

```diff
@@ -14,6 +14,7 @@ module Cardano.CLI.Byron.Delegation
 where
 
 import           Cardano.Api.Byron
```

Comment:
Would it be possible to use this through `cardano-api` ? In other words, remove dependency on `cardano-ledger-byron`